### PR TITLE
Update docs for `DOMNode::lookupNamespaceURI`

### PR DESCRIPTION
**Pull request by paulbalandan**
Ref:
- https://github.com/php/php-src/blob/c40dcf93d0b95d9a1026476b202f5aa965fb3fdd/ext/dom/node.c#L1622-L1650
- https://github.com/php/php-src/blob/b3553159a7b15d1ddd485aba9f83485f08e2e800/ext/dom/node.c#L1515-L1543
- https://www.w3.org/TR/DOM-Level-3-Core/core.html#Node3-lookupNamespaceURI

### CODE REVIEW

**Review by nielsdos — APPROVED** [review]
This is right, thank you!
Would like to get a second opinion about the wording, since I'm not very familiar with the wording used in docs. If no objections come in I can merge this.

File: reference/dom/domnode/lookupnamespaceuri.xml [hunk 1]
```diff
@@ -36,9 +36,30 @@
...
+     <row>
+      <entry>8.0.0</entry>
+      <entry>
+       The method is renamed to <function>lookupNamespaceURI</function>. Previously, this was <function>lookupNamespaceUri</function>.
+      </entry>
+     </row>
+    </tbody>
+   </tgroup>
+  </informaltable>
+ </refsect1>
```
[the remark this answers is not in the document]
Comment (paulbalandan) [reply reference/dom/domnode/lookupnamespaceuri.xml:62]: This info is useful for phpstan and phpstorm-stubs. In PHPStan, it checks for correct name casing. In phpstorm-stubs, there are two versions of the function based on the casing.

[the remark this answers is not in the document]
Comment (Girgias) [reply reference/dom/domnode/lookupnamespaceuri.xml:62]: External projects and tooling do not affect the official documentation. For an end-user calling either casing has _no_ effect, so having a changelog for this is pointless from my PoV.

[the remark this answers is not in the document]
Comment (paulbalandan) [reply reference/dom/domnode/lookupnamespaceuri.xml:62]:
```suggestion
```

[the remark this answers is not in the document]
Comment (paulbalandan) [reply reference/dom/domnode/lookupnamespaceuri.xml]: Done. Removed.

**Pull request merged**